src/main/scala/cromwell/Boot.scala

```diff
@@ -0,0 +1,65 @@
+package cromwell
+
+import java.io.File
+import java.nio.file.{Files, Paths}
+
+import akka.actor.{Props, ActorSystem}
+import akka.io.IO
+import akka.pattern.ask
+import akka.util.Timeout
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wordnik.swagger.model.ApiInfo
+import cromwell.engine.WorkflowManagerActor
+import cromwell.webservice._
+import spray.can.Http
+
+import scala.concurrent.duration._
+import scala.reflect.runtime.universe._
+import scala.util.{Failure, Success}
+
+
+object Boot extends App {
+
+  private def startup(): Unit = {
+    val conf = ConfigFactory.parseFile(new File("/etc/cromwell.conf"))
```

Comment:
yes -- that seems to be all the rage.  And when deployed as a container, the appropriate configuration gets poke into /etc